RevenueCatUI/CustomerCenter/Views/PurchaseHistory/PurchaseLinkView.swift

```diff
@@ -47,6 +48,7 @@ struct PurchaseLinkView: View {
                 .foregroundStyle(.secondary)
         }
         .contentShape(Rectangle())
+        .tint(.black)
```

Comment:
Is this correct? (asking regarding light/dark mode)

Reply:
| Dark | Light |
|--------|--------|
| <img width="542" alt="Screenshot 2025-01-28 at 17 59 47" src="https://github.com/user-attachments/assets/ff6ff07c-4c70-4072-abcc-29f70f1b4195" /> | <img width="542" alt="Screenshot 2025-01-28 at 18 00 06" src="https://github.com/user-attachments/assets/7322f643-12a5-4974-bd24-be5cab41da18" /> |

Reply:
Fixed! Nice catch @ajpallares ❤️